lib/rust/launcher-shims/src/lib.rs

```diff
@@ -0,0 +1,59 @@
+use std::env;
+use std::process::{Command, exit};
+use std::fs::OpenOptions;
+use std::io;
+use std::io::prelude::*;
+use std::path::PathBuf;
+
+pub fn wrap_launcher(version: &str) {
```

Comment:
You should depend on (and import) the `enso-prelude` such that you can take `version:impl Str` here.

Reply:
Hm, do you think it is really a good idea?

This function is designed to be only called with these constant strings and it's just a very simple wrapper. Adding prelude to it increases its build time from less than one second to a minute.

If I should add it, I guess the build should be done as a separate step on the CI to avoid hogging too much memory.

Reply:
Changed the types to `impl AsRef<str>`, I hope I did it correctly as I haven't been using Rust much before.